### DISCUSSION TITLE
Deciding on controlled vocabularies for inclusion in Invenio

### DISCUSSION THREAD

**Discussion by bjr70** [MESH-Research/knowledge-commons-repository#105] [General]
Continuing discussion from this issue: https://github.com/MESH-Research/knowledge-commons-repository/issues/16

I do want to keep the FOS vocabulary that came with Invenio and add MESH (medical subject headings). NASA (see below) looks good too. I also want to add Homosaurus - https://homosaurus.org/ - an international linked data vocabulary of Lesbian, Gay, Bisexual, Transgender, and Queer (LGBTQ) terms.

Links from the Northwestern meeting for plugins to some taxonomies:
- https://github.com/galterlibrary/invenio-subjects-lcsh/
- https://github.com/galterlibrary/invenio-subjects-mesh/

We use FAST, not LCSH, but I wonder if we could use that plugin as a model?

Lucas found the following when I asked about indigenous vocabularies and I'd like to consider these:
- New Zealand https://natlib.govt.nz/librarians/nga-upoko-tukutuku#about-the-project
- Australia https://www1.aiatsis.gov.au/ThesaurusDownload.html
- First Nations Metis and Inuit Indigenous Ontology (Canada) https://www.nikla-ancla.com/post/first-nations-metis-and-inuit-indigenous-ontologies-fnmiio (has a link to a Google sheet)

Not sure about this one. It's in Chinese. Could be helpful, but not sure what the implications would be.
- Ethnic group names (from Taiwan Central Library, in Chinese) https://catweb.ncl.edu.tw/subject2005/hs_appendix4.pdf

GND
- https://gnd.network/Webs/gnd/EN/FAQ/faq_node.html
- https://pypi.org/project/invenio-subjects-gnd/

NASA thesaurus
- https://sti.nasa.gov/nasa-thesaurus/#:~:text=The%20NASA%20Thesaurus%20contains%20over,contact%20the%20STI%20Information%20Desk%20.
- https://pypi.org/project/invenio-subjects-nasa/

It looks like the GND is in German? Not sure about that one, either.

If anyone is interested in controlled vocabularies BARTOC has a searchable database: https://bartoc.org/vocabularies?languages=en.

**Comment by bjr70** [comment]
There is a possible reason for including LCSH - US indigenous people and places, but there's been considerable discussion on how to decolonize the language included (see the bottom section): https://libguides.ucmerced.edu/native-american-indigenous/search-terms. I think it would be redundant to include with FAST, but FAST will have the same issues as it was created by LCSH. I would be inclined not to include it, but to perhaps somewhere in the documentation include a note that language used in some subject headings may not be terms preferred by indigenous populations.

**Comment by monotasker** [comment]
I've gone ahead and integrated the mesh vocabulary along with the fast terms. Yeah, I think it would be confusing for users if we were to use FAST and LCSH. They're going to get suggestions as they type from whatever vocabularies are installed. So there's going to be a balance, I think, between enough and too many.

**Reply by bjr70** [reply]
Yeah, agreed. I vote keep FAST and ditch LCSH.

**Reply by bjr70** [reply]
Along those lines I would not include the Chinese vocabulary or GND. I think we'll be covered by the others.

**Reply by monotasker** [reply]
Yeah, I'd agree

**Comment by monotasker** [comment]
The NASA thesaurus might be worth thinking about if we're supporting STEM fields. It apparently covers general physics, etc. as well as fields like astronomy. And it's an easy install. But I don't know what vocabularies (other than FOS) are standard in STEM, and I don't know whether it's necessary if we're including FOS.

**Reply by bjr70** [reply]
Hmmm FOS is expansive. Maybe we don't need NASA: https://explore.openaire.eu/fields-of-science. Mostly I just want to include it because I'm a space junkie. Probably redundant though.

We are bringing on STEMEd+ so we will need to support those disciplines.

**Comment by bjr70** [comment]
@mikethicke @babaklar @stephanievaskomsu any additional thoughts here?
